docs/community_resources/faq.md

```diff
@@ -0,0 +1,81 @@
+#FAQ
+
+This section contains answers for most common questions, tips around support and maintenance, references to important parts of the documentation and tools useful for developers on their daily basis.
+
+####What information should I specify when creating a Customer Support ticket?
+
+From the Customer Support perspective the most important thing is version of eZ Platform which is used in the project. The best way to specify it is providing the list of currently installed packages by running composer command:
+``` bash
+composer show ezsystems/*
+```
+
+Besides that all the configuration from `app/config` directory might be helpful. It is good to give steps to reproduce the issue, or at least some meaningful description of circumstances under which the problem occurred.
+
+If you stumbled upon some data inconsistency related problem providing database logs should also be important step.
+
+Also, please remember to mention recent changes, performed migrations or external scripts/code customizations related to the code which generates problem.
+
+####What are the recommended ways to increase my project's performance?
+
+The most important clues around increasing overall performance of your eZ Platform based project can be found in dedicated [documentation page](../guide/performance.md).
+
+####How can I translate my admin panel?
+
+The language of the admin panel is based on the browser language. In order to change it you should install proper package for you language, refer to the following [list](https://github.com/ezplatform-i18n). 
+Once you have language packages installed, you can switch the language of the Back Office in the User Settings menu.
+
+If you do not have a language defined in the browser, the language will be selected based on `parameters.locale_fallback` parameter located in `default_parameters.yml`.
+
+If you want to read more about languages managing in eZ Platform please see the following doc pages:
+
+- https://doc.ezplatform.com/en/latest/guide/internationalization/#back-office-languages
+- https://doc.ezplatform.com/en/latest/cookbook/setting_up_multi_language_siteaccesses/
+
+####How should I proceed with contributions to eZ Platform?
```

Comment:
covered, or should be covered on community_resources/contributing.md'?

Reply:
Partially yes, but it touches contributing to documentation. I mixed all the necessary steps from this one and from kernel metarepo README (both sources mentioned).

Reply:
Well, that page could be expanded to include the example here on how to contribute to code.

What we want to keep here is how to deal with patches from support, using composer-patches package. So a somewhat different topic.

Reply:
Code contribution procedure moved to: https://github.com/ezsystems/developer-documentation/pull/617

TODO: description of `composer-patches`.